cypress/integration/index.test.js

```diff
@@ -0,0 +1,43 @@
+describe("index", () => {
+  beforeEach(() => {
+    cy.visit("http://localhost:3000");
+  });
+
+  it("says 'Invalid input.' if pressed with no input", () => {
+    cy.get("button").click();
+  });
+
+  it("says 'Invalid input.' if pressed with invalid input", () => {
+    cy.get("input").type("a,b,c");
+    cy.get("button").click();
+  });
+
```

Comment:
The solution code also handles cases where the form is submitted but no option was selected from sum, average, or mode. Should there be a test to cover this as well? If yes, then that case should probably be mentioned in the README requirements as well.

Reply:
Oh actually I couldn't find a reasonable way to trigger that from the spec; I can just remove that logic.

Reply:
Works for me!